src/Surfnet/ServiceProviderDashboard/Domain/Entity/Entity/MetaData.php

```diff
@@ -257,6 +257,18 @@ private static function getAcsLocationsFromMetaDataFields(array $metaDataFields)
         return $acsLocations;
     }
 
+    private function asArrayAcsLocations(): array
+    {
+        $data = [];
+        foreach ($this->getAcsLocations() ?? [] as $index => $location) {
+            $location = sprintf('metaDataFields.AssertionConsumerService:%d:Location', $index);
+            $binding = sprintf('metaDataFields.AssertionConsumerService:%d:Binding', $index);
+            $data[$location] = $location;
+            $data[$binding] = Constants::BINDING_HTTP_POST;
+        }
+        return $data;
+    }
```

Comment:
Boyscout rule applied :medal_military: 

`$this->getAcsLocations() ?? []`: convenient solution to not having to wrap the foreach in an if block :+1: Might have to add this one to my own toolbelt